docs/usage/shoot_versions.md

```diff
@@ -0,0 +1,108 @@
+# Kubernetes and Machine image versions in Gardener
+
+## Motivation
+
+On the one hand-side Gardener is responsible for managing the Kubernetes and Machine image versions of its Shoot clusters.
+On the other hand-side, Gardener needs to be configured and updated based on the availability and support of the Kubernetes and Machine image version it provides.
+
+For instance the Kubernetes community releases **minor** versions roughly every three months and usually maintains **three minor** versions (the actual and the last two) with bug fixes and security updates. 
+Patch releases are done more frequently.
+
+Gardener needs to be configured accordingly to reflect the "logical state" of a version. 
```

Comment:
Could you point which is the default class for a version if omitted?

Reply:
I added a paragraph below  in ## Version Classifications